### PR TITLE
Implement reset zoom

### LINKED ISSUE #6521: Presentation menu - click to reset zoom to default
Clicking the current zoom level (center section in presentation zoom control) should reset zoom to default, consistent with other applications. (well, chrome doesn't do this, but firefox does)
Reported by @kepstin

### PR DESCRIPTION
**Pull request by vitormateusalmeida**
Closes #6521

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx [hunk 1]
```diff
@@ -162,22 +177,27 @@ export default class ZoomTool extends Component {
...
               className={styles.prevSlide}
               hideLabel
             />
           </HoldButton>
         ),
         (
-          <span
+          <Button
             key="zoom-tool-2"
+            role="button"
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx]: I noticed this in other spots in the code too - why do we specify `role="button"` to `Button`s? Don't they already have it?

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx [hunk 2]
```diff
@@ -162,22 +177,27 @@ export default class ZoomTool extends Component {
...
-            className={styles.zoomPercentageDisplay}
-          >
-            {`${this.state.value}%`}
-          </span>
+            aria-describedby={stateValue}
+            color="default"
+            customIcon={`${stateValue}%`}
+            size="md"
+            onClick={() => this.resetZoom()}
+            label="Reset Zoom"
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx]: Needs to be localized

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx (hunk 1 above)
Comment (jfsiebel) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/styles.scss [hunk 3]
```diff
@@ -122,4 +122,14 @@
   display: flex;
   align-items: center;
   justify-content: center;
+  border-top: 0 !important;
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/styles.scss]: Why are you using it !Important over all of these attributes?

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx (hunk 1 above)
Comment (capilkey) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx]: Why wouldn't the `role="button"` be set on the BaseButton?

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/styles.scss (hunk 3 above)
Comment (vitormateusalmeida) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/styles.scss]: This was necessary to override the default button style

**Comment by KDSBrowne** [comment]
I fixed those in https://github.com/bigbluebutton/bigbluebutton/pull/6660

**Comment by vitormateusalmeida** [comment]
I fixed too
https://github.com/bigbluebutton/bigbluebutton/pull/6720/commits/b62f78e9a97bfb0989b41d09b6e46f3bc142bd27

**Pull request merged**